src/yamlRestTest/resources/rest-api-spec/test/_plugins.ubi/20_queries_with_ubi.yml

```diff
@@ -6,7 +6,7 @@
         index: ecommerce
         body:
           mappings:
-            { "properties": { "category": { "type": "text" }, "timestamp": { "type": "date", "format": "strict_date_time" } } }
```

Comment:
did we want to remove this????   Didn't we just add in timestamp in a strict format?

Reply:
Yes, this was put here inadvertently because this is the mapping for the `ecommerce` index and not the `ubi_queries` index.

Reply:
ah!  thanks.